pkg/controller/command/didclient/command.go

```diff
@@ -149,9 +150,13 @@ func (c *Command) SaveDID(_ io.Writer, req io.Reader) command.Error {
 }
 
 func (c *Command) saveDID(didDataToStore *sdscomm.DIDDocData) command.Error {
-	sdsComm := sdscomm.New(c.sdsServerURL, c.agentUsername)
+	sdsComm, err := sdscomm.New(c.sdsServerURL, c.agentUsername)
```

Comment:
why are you creating new sdsComm instance everytime? looks like serverURL & agentUsername are always same here?

Reply:
Every time the go code executes it creates a new Command object. Is there a way I can avoid this?

Reply:
Are you sure it creates new command object for each call??
Still, you should move this to constructor(New()) function

Reply:
@sudeshrshetty Yeah, I actually did some testing since I was curious about exactly this... in fact, controller.GetCommandHandlers inside the addTrustBlocAgentHandlers function seemed to be called new every time I clicked...

Since this change seems unrelated to the current PR and will cause merge conflicts with the other PR, I'll fix this in a separate PR after my current two PRs are merged in. #267

Reply:
Clarification: I tested this with the Create DID button on the DID management page